src/test/scala/org/apache/spark/sql/pulsar/SchemaInfoSerDeSuite.scala

```diff
@@ -13,22 +13,27 @@
-    val ps1 = new SchemaInfo()
-    ps1.setName("datafromkafkatopulsar")
-    ps1.setSchema(s1.toString.getBytes(StandardCharsets.UTF_8))
-    ps1.setType(SchemaType.AVRO)
+    val ps1 = new SchemaInfoImpl("datafromkafkatopulsar",
```

Comment:
My major concern is that we don't have a test suite for each change. Did we run some smoke tests ?